### PR TITLE
Update various resource references

### PR DESCRIPTION
**Pull request by fouadhatem**
Update various resource references from IBM-Swift to Kitura, wherever applicable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Defaults to `false`. Specifies if decimal character escapes should be used inste
 ```swift
 import HTMLEntities
 
-let text = "한, 한, ế, ế, 🇺🇸"
+let text = "한, 한, ế, ế, 🇺🇸"
```
Comment (dannys42) [inline README.md:120]: Can you double-check these unicode strings?  I'm assuming they weren't intended to be changed?

Comment (fouadhatem) [reply README.md:120]: Yeah they weren't intended to be changed. Don't think I intentionally touched them. They don't even look changed to me. Do you see anything?

Comment (fouadhatem) [reply README.md:120]: That's strange. It appears in every line where these kinds of scalars exist. Could it be a Safari bug?

Comment (dannys42) [reply README.md:120]: They're definitely different.  Github is highlighting the changed values.  Did you update this from GitHub.com or by checking out and editing?  Probably need to just undo those changes with your editor.  (Xcode can do line-level undos if you need).

Comment (fouadhatem) [reply README.md:120]: I did those from a browser, so GitHub.com

Comment (fouadhatem) [reply README.md:120]: Oh they are different! I'll fix this.

Comment (fouadhatem) [reply README.md:120]: Hmm... I think something is wrong over at my end. Possibly a localization issue on my machine. Can you push through the other changes and ignore this file?

Comment (dannys42) [reply README.md:120]: If you're able to pull the repo locally, you can use Xcode to undo specific lines of changes, then re-push your PR.

**Pull request closed without merging**